content/docs/deployment/csminstallationwizard/src/index.html

```diff
@@ -109,11 +109,50 @@
                     </div>
                   </div>
 
+		  <div class="row mb-4 observability-operator">
+                    <div class="col-sm-12">
+                      <input class="form-check-input" type="checkbox" id="observability-operator" value="" onchange="onObservabilityOperatorChange()">
```

Comment:
Line number - 113 -132 can be removed and can reuse Line 100- 109 and onObservabilityChange() can updated to handle  onObservabilityOperatorChange()

Reply:
Will make these changes in separate PR .